components/layout/taffy/stylo_taffy/wrapper.rs

```diff
@@ -10,33 +10,41 @@ use super::convert;
 
 /// A wrapper struct for anything that Deref's to a [`ComputedValues`], which
 /// implements Taffy's layout traits and can used with Taffy's layout algorithms.
-pub struct TaffyStyloStyle<T: Deref<Target = ComputedValues>>(pub T);
+pub struct TaffyStyloStyle<T: Deref<Target = ComputedValues>> {
+    pub style: T,
+    pub is_replaced: bool,
+}
 
-impl<T: Deref<Target = ComputedValues>> From<T> for TaffyStyloStyle<T> {
-    fn from(value: T) -> Self {
-        Self(value)
+impl<T: Deref<Target = ComputedValues>> TaffyStyloStyle<T> {
+    pub fn new(style: T, is_replaced: bool) -> Self {
+        Self { style, is_replaced }
     }
 }
 
 impl<T: Deref<Target = ComputedValues>> taffy::CoreStyle for TaffyStyloStyle<T> {
     #[inline]
     fn box_generation_mode(&self) -> taffy::BoxGenerationMode {
-        convert::box_generation_mode(self.0.get_box().display)
+        convert::box_generation_mode(self.style.get_box().display)
     }
 
     #[inline]
     fn is_block(&self) -> bool {
-        convert::is_block(self.0.get_box().display)
+        convert::is_block(self.style.get_box().display)
+    }
+
+    #[inline]
+    fn is_compressible_replaced(&self) -> bool {
+        self.is_replaced
```

Comment:
This is a bit confusing since some elements that aren't actually replaced are considered to be compressible replaced.

Probably `TaffyStyloStyle::is_replaced` should be renamed to `TaffyStyloStyle::is_compressible_replaced`?
And in `get_grid_child_style()` add a TODO/FIXME comment.

Reply:
Renamed to `is_compressible_replaced`